run_kraken.py

```diff
@@ -34,6 +35,8 @@ def get_node_scenario_object(node_scenario):
         return gcp_node_scenarios()
     elif node_scenario['cloud_type'] == 'openstack':
         return openstack_node_scenarios()
+    elif node_scenario['cloud_type'] == 'bm':
+        return bm_node_scenarios(node_scenario["bm_user"], node_scenario["bm_password"])
```

Comment:
I think it might be a good idea to fail here if bm_uer or bm_password are empty.